### PR TITLE
updating installation instruction

### PR DESCRIPTION
**Pull request by e-krebs**
addressing issue #767 :
- adding more details in the installation instructions
- fixing the mentioned path which was wrong
- mentioning how to use `npm` instead of `yarn`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,20 +27,25 @@ This is only necessary when you want to build the extension yourself from source
...
-7. You can also use `yarn dev:chrome` to build & watch the unpacked extension.
+2. `cd vue-devtools` the newly created folder
+2. run `yarn install`
+3. then run `yarn run build`
+4. Open the Chrome extension page (currently under Menu > More Tools > Extensions)
+5. Check "developer mode" on the top-right corner
+6. Click the "load unpacked" button on the left, and choose the folder: `vue-devtools/packages/shell-chrome/`
+7. Alternatilvely to step 3, you can also use `yarn dev:chrome` to build & watch the unpacked extension
+
+When using Npm, just replace `yarn` with `npm`.
```
Comment (Akryum) [inline README.md]: Do not use npm

File: README.md [hunk 2]
```diff
@@ -27,20 +27,25 @@ This is only necessary when you want to build the extension yourself from source
...
 ### Development
 
 1. Clone this repo
-2. `yarn install`
-3. `yarn run dev`
+2. run `yarn install`
+3. then run `yarn run dev`
 4. A plain shell with a test app will be available at `localhost:8100`.
 
+When using Npm, just replace `yarn` with `npm`.
```
Comment (Akryum) [inline README.md]: Do not use npm

File: README.md [hunk 3]
```diff
@@ -62,7 +67,7 @@ This is only necessary when you want to build the extension yourself from source
 	$ yarn run run:firefox
 	~~~~
 
-	When using Yarn, just replace `npm` with `yarn`.
+	When using Npm, just replace `yarn` with `npm`.
```
Comment (Akryum) [inline README.md]: Same

Comment (e-krebs) [reply README.md]: should I remove the npm reference, then ?

Comment (Akryum) [reply README.md]: Yes

**Comment by e-krebs** [comment]
It makes sense.
I removed the reference to npm (including the previously existing one)

**Comment by Akryum** [comment]
👍

**Pull request merged**